src/Shared/Microsoft.DotNet.Internal.DependencyInjection.Testing/DependencyInjectionValidation.cs

```diff
@@ -14,6 +14,7 @@ public static class DependencyInjectionValidation
             "System.Fabric.ServiceContext",
             "System.Fabric.StatelessServiceContext",
             "System.Fabric.StatefulServiceContext",
+            "Microsoft.DotNet.ServiceFabric.ServiceHost.IServiceLoadReporter",
```

Comment:
Why does this get ignored?  It's some type we added... if something pulls it, and it's not registered by us, it's going to explode.

Reply:
is it because it happens in an unsuaul place that the tests don't account for?

Reply:
I added it, and it gets registered by the service implementations. It can't be registered by the AddDefaultServices thingy so I had to stick it here.